Dockerfile

```diff
@@ -17,7 +25,7 @@ USER root
 RUN mkdir var/logs && chown www-data:www-data var/logs
 
 USER www-data
-CMD ["venv/bin/python"]
```

Comment:
Docker containers should still use a virtualenv https://stackoverflow.com/a/33150800/91590 (whether it is through `pipenv` or `virtualenv` or another tool) to avoid interaction with the distribution packages; there's little else that run inside a container than the app itself, but `python` dependencies are surprisingly widespread in Linux images.

For example, you may find that installing a particular `requests` version in an app breaks [the system one](https://github.com/pypa/pipenv/blob/master/setup.py#L31), system tools then try to vendor it themselves (`pipenv` is doing this) shipping their own copy, in an arms race, etc.